app/models/video.py

```diff
@@ -0,0 +1,19 @@
+from app import db 
+
+
+class Video(db.Model):
+    __tablename__ = "videos"
+    video_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime)
+    total_inventory = db.Column(db.Integer)
+    available_inventory = db.Column(db.Integer)
```

Comment:
Like with `Customer` you can compute `available_inventory` with a relationship:

```suggestion
    rentals = db.relationship('Rental', back_populates='video', lazy=True)
```